QuizWhois.Domain/Services/Implementations/QuestionService.cs

```diff
@@ -30,5 +30,33 @@ public QuestionModel AddQuestion(QuestionModel operationModel)
             _context.SaveChanges();
             return new QuestionModel(entity.Id, entity.QuestionText, entity.CorrectAnswer);
         }
+
+        public QuestionModel GetRandomQuestion()
+        {
+            var random = new Random().Next(0, _context.Set<Question>().Count());
+            var randomRecord = _context.Set<Question>().OrderBy(x => x.Id).Skip(random).FirstOrDefault();
+            if (randomRecord is null)
+            {
+                throw new Exception("GetRandomQuestion in QuestionService.GetRandomQuestion was null");
+            }
+            else
+                return new QuestionModel(randomRecord.Id, randomRecord.QuestionText, randomRecord.CorrectAnswer);
+        }
+
+        public bool CheckAnswer(QuestionModel operationModel)
+        {
+            if (operationModel == null || operationModel.QuestionText == string.Empty || operationModel.CorrectAnswer == string.Empty)
```

Comment:
it is really confusing to name users answer correct before the check

Reply:
i will create a model which have userAnswer instead of correctAnswer